src/converse-muc-views.js

```diff
@@ -693,6 +697,17 @@
                         msgid
                     });
                 },
+                sendChatRoomFile (text) {
+                    const msgid = _converse.connection.getUniqueId();
+                    const stanza = $msg({
+                        'from': _converse.connection.jid,
+                        'to': this.model.get('jid'),
+                        'type': 'groupchat',
+                        'id': msgid
+                    }).c("body").t(text).up()
+                      .c("x", {'xmlns': Strophe.NS.OUTOFBAND}).c('url').t(text).up();
+                     _converse.connection.send(stanza);
+                },
```

Comment:
This should to onto the `_converse.ChatRoom` model in `converse-muc.js` for the same reasons as mentioned above.